test/opt/copy_prop_array_test.cpp

```diff
@@ -1839,6 +1839,81 @@ OpFunctionEnd
 
   SinglePassRunAndCheck<CopyPropagateArrays>(text, text, false);
 }
+
+TEST_F(CopyPropArrayPassTest, EntryPointUser) {
```

Comment:
This is an old way of doing tests.  Can you change this to a match test:

https://github.com/KhronosGroup/SPIRV-Tools/blob/0a43a84e02245cca40ce187d1e427a5d0b4f3d13/test/opt/copy_prop_array_test.cpp#L28

They are better at calling out the important part of the test.

Also add a comment explaining the purpose of the test.  Something like:

> Since Spir-V 1.4, resources that are used by a shader must be on the OpEntryPoint instruction with the inputs and outputs.  This test ensures that this does not stop the pass from working.